src/stylesheets/components/_footer.scss

```diff
@@ -1,29 +1,17 @@
-body > {
-  footer {
-    width: 100%;
-    display: inline-block;
-    box-sizing: border-box;
-    background-color: $deep-purple;
-    padding-bottom: 60px;
-  }
+/*--- Global Footer ---*/
+
+body > footer {
+  width: 100%;
+  display: inline-block;
+  box-sizing: border-box;
+  background-color: $deep-purple;
+  padding-bottom: 60px;
 
-  /*--- Print Styles ---*/
+  /* Print Styles */
 
   @media print {
     footer {
```

Comment:
This is incorrect, the global footer should be omitted from print, therefore remove footer declaration within the print media query

Reply:
Correct